eth-connector/src/admin_controlled.rs

```diff
@@ -84,3 +84,40 @@ pub mod error {
         }
     }
 }
+
+#[test]
+fn test_pause_control() {
```

Comment:
Should this be in a `#[cfg(test)]` module?

Reply:
It seems not necessary in this case. This code won't be included in the binary file created by `cargo build` or `cargo install` commands. I think creating a `#[cfg(test)]` module makes sense if you have some additional artifacts used in tests only.